scripts/apps/search/services/SearchService.js

```diff
@@ -516,6 +516,23 @@ export function SearchService($location, gettext, config, session) {
     };
 
     /**
+     * Merges the highlighted fields to the item
```

Comment:
I know it's now confusing with many types of documentation syntaxes throughout the project. I'm working on evening it all out and converting it, but in the meantime, we'll need help! Please check using `grunt docs`. I think you need to use the format that is used further down in this file on `this.doesSearchAgainstRepo`. Do you mind correcting?